src/utils/CommonUtils.js

```diff
@@ -11,8 +11,8 @@ class CommonUtils {
 		}
 
 		let d     = new Date(),
-		    dash  = "-",
-		    colon = ":";
+			dash  = "-",
```

Comment:
Get rid of purely space changes, it pollutes the commit

Reply:
true, will be more carefull next time. but can we now leave this one? this way this file stays all consistent